hangman.js

```diff
@@ -1 +1,21 @@
-console.log("welcome to hangman");
\ No newline at end of file
+console.log("welcome to hangman");
+var word = prompt("Enter a word.");
+console.log("word:" + word);
+var dashWord = "-";
+for (i = 1; i <word.length; i ++) {
+  dashWord += "-";
+}
+// var dashWord = "-----";
```

Comment:
the above logic is great! I recommend putting this into a function so you can show letters after a user has guessed correctly, e.g.:

```
function dashWord(word, guesses){
  var dashes = "";
  for(var i = 0 ; i < word.length; i++ ){
    if(guesses.includes(word[i])){
      dashes += word[i]
    } else {
      dashes += "-"
    }
  }
}
```